src/components/navigation/navigation.spec.js

```diff
@@ -86,6 +86,66 @@ const EXAMPLE_NAVIGATION_WITH_SUBNAVIGATION = {
   },
 };
 
+const EXAMPLE_NAVIGATION_WITH_HIDDEN_SUBNAVIGATION = {
```

Comment:
Would be good to rename this because the sub navigation isn't being hidden its being removed